### PR TITLE
[develop] Base Project

### PR DESCRIPTION
**Pull request by julia-jpg**
add initial files

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,7 @@
+baseUrl=
+token=
+username=
+password=
+schemasPath=src/test/resources/schemas/
+filterTags=
+cucumberThreadCount=1
```
Comment (MirkoFer-622) [inline gradle.properties:9]: In the specification of the task, it is necesary to define these others properties:
clientId=
clientSecret=
keyToken=

Please, consider add them.

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1,7 @@
+baseUrl=
+token=
```
Comment (MirkoFer-622) [inline gradle.properties]: Please, delete "token" property.

File: src/main/resources/Log4j2.properties [hunk 3]
```diff
@@ -0,0 +1,19 @@
+property.filename = logs
+appenders = console, file
+appender.console.type = Console
+appender.console.name = STDOUT
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = [%-5level] %d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %c{1} - %msg%n
+
+appender.file.type = File
+appender.file.name = LOGFILE
+appender.file.fileName=${filename}/pivotal.log
```
Comment (MirkoFer-622) [inline src/main/resources/Log4j2.properties]: You should change the directory named as "pivotal.log" to "salesforce.log"

**Review by MirkoFer-622 — COMMENTED** [review]
Runner class was not created, please consider do it.

**Review by MirkoFer-622 — APPROVED** [review]
It looks good, it's ready to merge!

File: build.gradle [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    useTestNG()
+    options {
+        useDefaultListeners = false
+        jvmArgs = ['-Dcucumber.plugin=io.qameta.allure.cucumber6jvm.AllureCucumber6Jvm']
+        jvmArgs = ['-Dcucumber.plugin=rerun:build/target/rerun.txt']
+        jvmArgs = ['-Dcucumber.filter.tags=' + filterTags]
+        jvmArgs = ['-DcucumberThreadCount=' + cucumberThreadCount]
+        jvmArgs = ['-Dusername=' + username]
+        jvmArgs = ['-Dpassword=' + password]
+    }
```
Comment (marcos-x86) [inline build.gradle:105]: Please consider adding the remaining variables for authentication.

File: build.gradle [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+    useTestNG()
+    options {
+        useDefaultListeners = false
+        jvmArgs = ['-Dcucumber.plugin=io.qameta.allure.cucumber6jvm.AllureCucumber6Jvm']
+        jvmArgs = ['-Dcucumber.plugin=rerun:build/target/rerun.txt']
+        jvmArgs = ['-Dcucumber.features=@build/target/rerun.txt']
+        jvmArgs = ['-DcucumberThreadCount=' + cucumberThreadCount]
+        jvmArgs = ['-Dusername=' + username]
+        jvmArgs = ['-Dpassword=' + password]
+    }
```
Comment (marcos-x86) [inline build.gradle:124]: Please consider adding the remaining variables for authentication.

**Review by marcos-x86 — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 4 above)
Comment (MirkoFer-622) [reply build.gradle:105]: The variables were updated at last commit

File: build.gradle (hunk 5 above)
Comment (MirkoFer-622) [reply build.gradle:124]: The variables were updated at last commit too

**Review by MirkoFer-622 — APPROVED** [review]

**Review by marcos-x86 — APPROVED** [review]
LGMT!

**Pull request merged**